input_writer.go

```diff
@@ -0,0 +1,30 @@
+package logrus
+
+import (
+	"bufio"
+	"io"
+	"runtime"
+)
+
+func (logger *Logger) InputWriter() (*io.PipeWriter) {
```

Comment:
The reason I called it `InputWriter` was because of ambiguity. I felt that as just `Writer`, it was unclear if this was something logrus writes to, or something for writing to logrus.
But it's your project, I'll change it :-)